files/build_templates/sonic_debian_extension.j2

```diff
@@ -759,6 +760,17 @@ sudo LANG=C DOCKER_HOST="$DOCKER_HOST" chroot $FILESYSTEM_ROOT docker tag {{imag
 fi
 {% endfor %}
 
+if [[ $CONFIGURED_PLATFORM == pensando ]]; then
+#we need to start pciemgrd process soon at the bootup for pci txn to be up.
+#if we don't start pciemgrd early pcie timeout happens and server reboots.
+#pciemgrd runs as part of dpu docker. So load this at build time itself.
+sudo LANG=C DOCKER_HOST="$DOCKER_HOST" chroot $FILESYSTEM_ROOT docker load -i boot/dpu_sw.tar
+sudo LANG=C chroot $FILESYSTEM_ROOT rm -f boot/dpu_sw.tar
+
+#Disable rc.local
```

Comment:
why do we need to disable rc.local here?

Reply:
We are seeing multiple errors on console with rc.local enabled. Also we don't need most of the functionality done in that script. 

We will disable in this initial change and revisit this later.

Reply:
We are seeing continuous error messages as below. So disabled it for now. It can be enabled later
```
[ 1561.958120] rc.local[18331]: + SONIC_VERSION=dpu.0-dirty-20230922.131734
[ 1561.958192] rc.local[18331]: + FIRST_BOOT_FILE=/host/image-dpu.0-dirty-20230922.131734/platform/firsttime
[ 1561.958261] rc.local[18331]: + SONIC_CONFIG_DIR=/host/image-dpu.0-dirty-20230922.131734/sonic-config
[ 1561.958329] rc.local[18331]: + SONIC_ENV_FILE=/host/image-dpu.0-dirty-20230922.131734/sonic-config/sonic-environment
[ 1561.958399] rc.local[18331]: + [ -d /host/image-dpu.0-dirty-20230922.131734/sonic-config -a -f /host/image-dpu.0-dirty-20230922.131734/sonic-config/sonic-environment ]
[ 1561.958467] rc.local[18331]: + logger SONiC version dpu.0-dirty-20230922.131734 starting up...
[ 1561.958536] rc.local[18331]: + grub_installation_needed=
[ 1561.958613] rc.local[18331]: + [ ! -e /host/machine.conf ]
[ 1561.958825] rc.local[18320]: + blkid
[ 1561.959095] rc.local[18338]: + blkid
[ 1561.959343] rc.local[18321]: + grep ONIE-BOOT
[ 1561.959636] rc.local[18339]: + grep ONIE-BOOT
[ 1561.959882] rc.local[18322]: + head -n 1
[ 1561.960187] rc.local[18340]: + head -n 1
[ 1561.960426] rc.local[18323]: + awk {print $1}
[ 1561.960800] rc.local[18341]: + awk {print $1}
[ 1561.961066] rc.local[18324]: + sed -e s/:.*$//
[ 1561.961401] rc.local[18342]: + sed -e s/:.*$//
[ 1561.961534] rc.local[18313]: + onie_dev=
[ 1561.961611] rc.local[18313]: + mkdir -p /mnt/onie-boot
[ 1561.961685] rc.local[18313]: + mount /mnt/onie-boot
[ 1561.961916] rc.local[18331]: + onie_dev=
[ 1561.961997] rc.local[18331]: + mkdir -p /mnt/onie-boot
[ 1561.962070] rc.local[18331]: + mount /mnt/onie-boot
[ 1561.962295] rc.local[18326]: mount: /mnt/onie-boot: can't find in /etc/fstab.
[ 1561.962639] rc.local[18344]: mount: /mnt/onie-boot: can't find in /etc/fstab.
[ 1561.962833] rc.local[18313]: + onie_grub_cfg=/mnt/onie-boot/onie/grub/grub-machine.cfg
[ 1561.962908] rc.local[18313]: + [ ! -e /mnt/onie-boot/onie/grub/grub-machine.cfg ]
[ 1562.304038] rc.local[18313]: + log_migration /mnt/onie-boot/onie/grub/grub-machine.cfg not found
[ 1562.304128] rc.local[18313]: + echo /mnt/onie-boot/onie/grub/grub-machine.cfg not found
[ 1562.304200] rc.local[18313]: + migrate_nos_configuration
[ 1562.304270] rc.local[18313]: + rm -rf /host/migration
[ 1562.304338] rc.local[18313]: + mkdir -p /host/migration
[ 1562.304822] rc.local[18331]: + onie_grub_cfg=/mnt/onie-boot/onie/grub/grub-machine.cfg
[ 1562.304911] rc.local[18331]: + [ ! -e /mnt/onie-boot/onie/grub/grub-machine.cfg ]
[ 1562.304994] rc.local[18331]: + log_migration /mnt/onie-boot/onie/grub/grub-machine.cfg not found
[ 1562.305063] rc.local[18331]: + echo /mnt/onie-boot/onie/grub/grub-machine.cfg not found
[ 1562.305132] rc.local[18331]: + migrate_nos_configuration
```